Added grit transformations to apply python lints (#12)
getgrit/python: .grit/patterns/aware_utc.md

---

> title: Aware DateTime for UTC

Reviewer: ```suggestion
title: Prefer timezone-aware datetimes to utcnow()
```

---

> To get the current time in UTC use an aware datetime object with the timezone explicitly set to UTC.

Reviewer: ```suggestion
To get the current time in UTC use a datetime object with the timezone explicitly set to UTC.
```
